src/main/java/com/netflix/blitz4j/LoggingConfiguration.java

```diff
@@ -347,11 +363,9 @@ public synchronized void reconfigure(Properties props) {
         // set of original initialization properties
```

Comment:
Mention in the method's javadoc that the method assumes the Properties object is pre-filtered!

Reply:
Updated the javadoc to include this assumption